src/main/java/biblioteka/Calendar.java

```diff
@@ -0,0 +1,354 @@
+            e.printStackTrace();
+        }
+        createTables();
+    }
+
+    public boolean createTables()
+    {
+        String createUsers = "CREATE TABLE IF NOT EXISTS users(user_id serial primary key, name varchar, surname varchar)";
+        //String createUsers = "CREATE TABLE IF NOT EXISTS users(name varchar, surname varchar)";
+        String createThings = "CREATE TABLE IF NOT EXISTS things(thing_id serial primary key, name varchar)";
+
+        try {
+            stat.execute(createUsers);
+            stat.execute(createThings);
+        } catch (SQLException e) {
+            System.err.println("Error during creating table");
+            e.printStackTrace();
+            return false;
+        }
+        return true;
+    }
+
+    public long insertUser(User user) {
+
+        String SQL = "INSERT INTO users(name, surname) "
+                + "VALUES(?,?)";
+
+        long id = 0;
+
+        try {PreparedStatement pstmt = conn.prepareStatement(SQL, Statement.RETURN_GENERATED_KEYS);
+            pstmt.setString(1, user.getName());
+            pstmt.setString(2, user.getSurname());
+
+            int affectedRows = pstmt.executeUpdate();
+            // check the affected rows
+            if (affectedRows > 0) {
+                // get the ID back
+                try (ResultSet rs = pstmt.getGeneratedKeys()) {
+                    if (rs.next()) {
+                        id = rs.getLong(1);
+                    }
+                }catch (SQLException ex) {
+                    System.out.println(ex.getMessage());
+                }
+            }
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+        return id;
+    }
+
+    public void insertUsers(List<User> list) {
+        String SQL = "INSERT INTO users(name, surname) "
+                + "VALUES(?,?)";
+        try (
+                PreparedStatement statement = conn.prepareStatement(SQL);) {
+            int count = 0;
+
+            for (User user : list) {
+                statement.setString(1, user.getName());
+                statement.setString(2, user.getSurname());
+
+                statement.addBatch();
+                count++;
+                // execute every 100 rows or less
+                if (count % 100 == 0 || count == list.size()) {
+                    statement.executeBatch();
+                }
+            }
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public int getUserCount() {
+        String SQL = "SELECT count(*) FROM users";
+        int count = 0;
+
+        try (Statement stmt = conn.createStatement();
+             ResultSet rs = stmt.executeQuery(SQL)) {
+            rs.next();
+            count = rs.getInt(1);
+        } catch (SQLException ex) {
+            System.out.println(ex.getMessage());
+        }
+
+        return count;
+    }
+
+    public void findUserByID(int userID) {
```

Comment:
This method should return user as promised in the method's name

Reply:
Working on it